woocommerce/rest-api/Controllers/Settings.php

```diff
@@ -155,7 +156,44 @@ public function update_item_permissions_check( $request ) {
 	/** Utility methods ***********************************************************************************************/
 
 
-	// TODO: prepare_item_for_response()
+	/**
+	 * @sincex.y.z
+	 *
+	 * @param Setting $setting a setting object
+	 * @param \WP_REST_Request $request request object
+	 * @return WP_Error|WP_HTTP_Response
+	 */
+	public function prepare_item_for_response( $setting, $request ) {
+
+		if ( $setting instanceof Setting ) {
+
+			$item = [
+				'id'          => $setting->get_id(),
+				'type'        => $setting->get_type(),
+				'name'        => $setting->get_name(),
+				'description' => $setting->get_description(),
+				'is_multi'    => $setting->is_is_multi(),
+				'options'     => $setting->get_options(),
+				'default'     => $setting->get_default(),
+				'value'       => $setting->get_value(),
+			];
+
+			if ( $control = $setting->get_control() ) {
+				$item['control'] = [
```

Comment:
I think it may be better for consistency to have the control key with a null value then not to have it.

Reply:
Sounds good. I changed that in 6249c31 and 5499ec1